parent/pom.xml

```diff
@@ -52,6 +53,12 @@
     <dependencyManagement>
         <dependencies>
 
+            <dependency>
+                <groupId>commons-io</groupId>
+                <artifactId>commons-io</artifactId>
+                <version>${commons-io.version}</version>
```

Comment:
nit: stick embedded the commons-io version directly here, as it's only used once, and add a comment, so it's clear it can/should be removed in the future